.github/workflows/dependabot-auto-label.yml

```diff
@@ -0,0 +1,29 @@
+# Source: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#label-a-pull-request
+name: Dependabot auto-label
+# Can't use `pull_request` due to GITHUB_TOKEN and security
+# See: https://github.com/dependabot/dependabot-core/issues/3253#issuecomment-852541544
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  issues: write
+  repository-projects: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
```

Comment:
Don't we prefer pinned versions?

Reply:
Ack, sorry, i overzealously merged while on the call 🙃